.markdownlintrc

```diff
@@ -0,0 +1,11 @@
+{
+  "default": true,
+  "line_length": false,
+  "single-title": "",
+  "no-inline-html": {
+    "allowed_elements": ["p", "img"] 
+  },
+  "no-trailing-punctuation": {
+    "punctuation": ",;:!。，；：！"
+  }
+}
```

Comment:
can we _not_ put this in the repo root?

Reply:
somewhere deep in hack/, where it's not cluttering discovery of the core repo content